docker-compose.yml

```diff
@@ -1,26 +1,26 @@
 version: '3'
 services:
-  movieinfo:
-    image: "keremk/movieinfo:v0.3"
-    ports:
-    - "3030:3030"
-    environment:
-    - FAIL_PERCENT=0.2
-    - MAX_ALLOWED=5
-  moviesearch:
-    image: "keremk/moviesearch:v0.4"
-    ports:
-    - "3040:3030"
-    environment:
-    - FAIL_PERCENT=0.1
-  artistinfo:
-    image: "keremk/artistinfo:v0.3"
-    ports:
-    - "3050:3030"
-    environment:
-    - FAIL_PERCENT=0.5
-    - MAX_ALLOWED=5
-  ratingsstream:
-    image: "keremk/ratingsstream:v0.3"
-    ports:
-    - "3020:3000"
\ No newline at end of file
+ movieinfo:
+   image: "keremk/movieinfo:v0.3"
+   ports:
+   - "3030:3030"
```

Comment:
need tabs:  YAML files is following this model: 
```services:
    web:
        image: app:web-1.0.0
        ports:
            - "8080:80"
        links:
            - php

```

Reply:
reformatted